packages/headless/src/features/instant-results/instant-results-slice.ts

```diff
@@ -52,6 +52,7 @@ export const instantResultsReducer = createReducer(
 
       const cached = getCached(state, action.meta);
       cached!.isLoading = true;
+      cached!.isActive = true;
```

Comment:
Technically should be active before you get the response, matters for the queryText too which is retrieved here